crates/astria-sequencer-relayer/local.env.example

```diff
@@ -0,0 +1,45 @@
+# Address of sequencer/cometbft/tendermint to request new blocks.
+# 127.0.0.1:26657 is the default socket address at which cometbft
+# serves RPCs.
+ASTRIA_SEQUENCER_RELAYER_SEQUENCER_ENDPOINT="http://127.0.0.1:26657"
+
+# Address of the celestia node. 127.0.0.1:26658 is the default socket
+# address at which celestia node servces RPCs.
+ASTRIA_SEQUENCER_RELAYER_CELESTIA_ENDPOINT="http://127.0.0.1:26658"
```

Comment:
this conflicts with the sequencer app port :// (this is the port ABCI applications use by default). we should probably change the default celestia endpoint?

Reply:
Right, so 26658 is even wrong. It's 26657 actually, but that clashes with cometbft's port at `:26657`.

To be honest, all of this is kind of an example and not a recommendation for actual ports to use. It should still be on the person deploying these services to wire them up correctly.